src/shared/resources/myinfo/index.ts

```diff
@@ -49,7 +49,7 @@ interface IMyInfoFieldType {
   description: string
   fieldType: BasicField
   fieldOptions?: string[]
-  ValidationOptions?: object
+  ValidationOptions?: Record<string, unknown>
```

Comment:
could you explain this?

Reply:
`object` is discouraged, use `Record<string, unknown>` to type instead.

> Avoid the object type, as it is currently hard to use due to not being able to assert that keys exist.
See microsoft/TypeScript#21732.